src/entrypoint.sh

```diff
@@ -2,10 +2,11 @@
 #
 # Entrypoint for Docker.
 
-readonly CONFIGURATION_FILE="${1}"
-readonly TOKEN="${2}"
+export RENOVATE_TOKEN="${2}"
```

Comment:
I think it would be better to prepend `RENOVATE_TOKEN="${2}"` to `/usr/local/bin/docker-entrypoint.sh` rather then exporting it. I'm not 100% sure about this, but think when you prepend it, only that command/subshell can access that env var. And with the export we risk exposing the token.

Reply:
Can you please propose the changes, I'm on mobile. Or simply push you changes and merge. I'm happy with that.